jvm/src/main/scala/com/intel/oap/GazellePluginConfig.scala

```diff
@@ -25,22 +25,22 @@ case class GazelleNumaBindingInfo(
     totalCoreRange: Array[String] = null,
     numCoresPerExecutor: Int = -1) {}
 
-class GazellePluginConfig(conf: SQLConf) extends Logging {
+class GazelleJniConfig(conf: SQLConf) extends Logging {
```

Comment:
Change the file name to GazelleJniConfig.scala too?

Reply:
Fixed.